docs/modeling-your-data/modeling-your-data-with-dbt/index.md

```diff
@@ -67,5 +68,12 @@ The supported data warehouses per version can be seen below:
 | ----------------------------- | -------------------- | ----------------- | -------- | ---------- | -------- | --------- | -------- |
 | 0.1.0                         | >=0.9.0 to <1.0.0    | >=1.0.0 to <2.0.0 | ✅        | ✅        | ❌        | ✅        | ❌        |
 
+</TabItem>
+<TabItem value="ecommerce" label="Snowplow Ecommerce">
+
+| snowplow-normalize version | snowplow-web version | dbt versions      | BigQuery | Databricks | Redshift | Snowflake | Postgres |
+| ----------------------------- | -------------------- | ----------------- | -------- | ---------- | -------- | --------- | -------- |
+| 0.1.0                         | >=0.9.0 to <1.0.0    | >=1.0.0 to <2.0.0 | ✅        |  ❌       | ❌        | ❌        | ❌        |
```

Comment:
You'll require 1.3 won't you?

Reply:
Not yet, built on 1.2 and utils v0.12 on purpose